### DISCUSSION TITLE
Restarting from my restart file

### DISCUSSION THREAD

**Discussion by dewan1988** [idaholab/moose#25518] [Q&A General]
Dear moose experts

I am a new user of MOOSE. I am trying to use a restarting command to start my simulation from the previous time steps. 
restart_file_base = out_cp/LATEST
checkpoint = true
it's showing that the simulation started from the previous time steps (1st image) but in the printing value it's showing that it is started from the beginning again. Could you please help me regarding these issues.

![image](https://github.com/idaholab/moose/assets/145499408/e1ffb4c9-1867-47db-baf9-f70af0b99a72)

![image](https://github.com/idaholab/moose/assets/145499408/96ea4582-13b9-4ec0-b517-903fcdade9da)

**Comment by GiudGiud** [comment]
Hello

the value of the start-time is determined by the input file, not the restart file iirc. A recover instead of a restart would use the checkpoint time
You ll want to change the start time in the executioner

Guillaume

**Reply by dewan1988** [reply]
Thank you for your kind reply. What should I do now? Actually, I want to restart my file from the previous time steps (i.e., time step:3). Should I change the starting time in the executioner? For example, if my code run 30 minutes initially and passed 3-time steps then should I use the 30 minutes as the staring time to restart my simulation from the time step 3.

**Reply by GiudGiud** [reply]
If you specify the LATEST in the restart file base parameter of the problem, you are restarting from the latest step, which is 3 right?
When it says "Using ..." it s actually loading that file into each variable. 

If you want the output to say time = 30min on that first step (in the console log), then you just need to change the start_time to make that happen

**Reply by dewan1988** [reply]
Yes, it shows that it is starting from time step 3 but in the printing output it is showing that it is started from time step zero (image-2). How can I modify the input file that my printing output will show that it is started from the time step 3?

**Reply by GiudGiud** [reply]
you can change the start_time in the Executioner block to change the time step output
